test/components/connect.spec.js

```diff
@@ -2009,7 +2008,7 @@ describe('React', () => {
 
       @connect(null, mapDispatchFactory, mergeParentDispatch)
       class Passthrough extends Component {
-        componentWillUpdate() {
+        componentDIdUpdate() {
```

Comment:
There is a type here? componentD*I*dUpdate

Reply:
Will fix & add test